src/redux/store/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from "@reduxjs/toolkit";
+import loginSlice from "../reducers/loginSlice";
+import thunk from "redux-thunk";
```

Comment:
we don't need to download this thunk
redux toolkit provide thunk by default
search how to get default middlewares and use this